engine/lib/metastrings.php

```diff
@@ -284,6 +279,14 @@ function _elgg_get_metastring_based_objects($options) {
 		$query = "SELECT {$options['metastring_calculation']}(v.string) as calculation FROM {$db_prefix}$type n_table";
 	}
 
+	foreach ($joins as $i => $join) {
+		if ($join === false) {
```

Comment:
Any idea what this does and why we have it? Looks odd and not commented. Just asking.